northstar/src/runtime/island/mod.rs

```diff
@@ -107,10 +112,20 @@ impl Launcher for Island {
     where
         Self: Sized,
     {
-        Ok(Island { tx, config })
+        let (tripwire_read, tripwire_write) =
```

Comment:
This causes the fd to automatically be closed during execve, right? Why do we need this for our tripwire?

Reply:
We safe an explicit `close` due it should also happen on `drop` of `PipeWrite`.